.gitignore

```diff
@@ -40,7 +40,7 @@ MANIFEST
 pip-log.txt
 pip-delete-this-directory.txt
 
-# Unit test.py / coverage reports
+# Unit test_file.py / coverage reports
```

Comment:
I think they are literally just saying 'unit test' so probably don't need the `.py` part here. As you can see we're just excluding a few rando folders